website/static/js/contribRemover.js

```diff
@@ -0,0 +1,312 @@
+/**
+ * Controller for the Remove Contributor modal.
+ */
+'use strict';
+
+var $ = require('jquery');
+var ko = require('knockout');
+var bootbox = require('bootbox');
+var Raven = require('raven-js');
+
+var oop = require('./oop');
+var $osf = require('./osfHelpers');
+var Paginator = require('./paginator');
+
+function getNodesOriginal(nodeTree, nodesOriginal) {
```

Comment:
Or maybe it belongs in [`fanghorn.js`](https://github.com/CenterForOpenScience/osf.io/blob/336d249199693b5073842d97f74d12ca74ac496b/website/static/js/fangorn.js)? Or treebeard itself?